geo/src/algorithm/contains/polygon.rs

```diff
@@ -12,6 +12,12 @@ where
     T: HasKernel,
 {
     fn contains(&self, coord: &Coordinate<T>) -> bool {
+        // Officially, we require the exterior to be a valid ring, but this isn't curently enforced
+        // by the Polygon contructor. We bail out here rather than trigger later debug asserts.
+        if self.exterior().0.is_empty() {
+            return false;
```

Comment:
This is to avoid hitting the assert in `coord_pos_relative_to_ring`.

Without this, a unit test exercises that path and fails.

There's arguably other approaches - like deleting that unit test. I think ultimately a more blessed path for validating geometries or perhaps having Result types that return Err where we're currently asserting. But I didn't particularly feel like taking that larger change on yet.